nexus/src/external_api/http_entrypoints.rs

```diff
@@ -1542,6 +1543,8 @@ async fn disk_metrics_list(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+/// Start importing blocks into a disk
+///
 /// Start the process of importing blocks into a disk
```

Comment:
The title and body here say the same thing in a slightly different way. Do we need them both?